src/app/pages/storage/components/dashboard-pool/storage-health-card/storage-health-card.component.html

```diff
@@ -55,12 +55,15 @@ <h3 mat-card-title>
   <mat-card-content class="card-content">
     <div class="details-item">
       <div class="label">{{ 'Pool Status' | translate }}:</div>
-      <div class="value">{{ pool().status | mapValue: poolStatusLabels | translate }}</div>
-    </div>
+      <div class="value font-bold">
```

Comment:
This should be just one line without `Pool Status` label.

When errors are present, you can still show them without `Total ZFS Errors:` text.

Example states:
```
Online, no errors.
Degraded, 5 errors.
```